### PR TITLE
Add error sensor to Husqvarna Automower

### PR DESCRIPTION
**Pull request by Thomas55555**
## Proposed change

Add documentation for the error sensor

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/113165
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by gjohansson-ST — CHANGES_REQUESTED** [review]
Missing `restricted_reason`

File: source/_integrations/husqvarna_automower.markdown [hunk 1]
```diff
@@ -94,6 +94,7 @@ The integration will create the following sensors:
 
 - Battery level
 - Cutting blade usage time (if available)
+- Error
```
Comment (c0ffeeca7) [inline source/_integrations/husqvarna_automower.markdown]:
```suggestion
- Error. For example: *Mower tilted*, *outside geofence*.
- Restricted reason. For example: *Week schedule*, *frost*, or *daily limit*.
```

**Comment by c0ffeeca7** [comment]
> Missing `restricted_reason`

Added restricted reason to move this along before the release. Do you approve, @gjohansson-ST ?

**Review by gjohansson-ST — APPROVED** [review]
Thanks @Thomas55555 and @c0ffeeca7 👍

**Pull request merged**